Source/ParameterEncoding.swift

```diff
@@ -310,5 +318,7 @@ public struct JSONEncoding: ParameterEncoding {
 // MARK: -
 
 extension NSNumber {
+    #if !os(Linux)
     fileprivate var isBool: Bool { return CFBooleanGetTypeID() == CFGetTypeID(self) }
```

Comment:
I don't think emulating this will be possible on Linux, as [the Swift team](https://github.com/apple/swift-evolution/blob/master/proposals/0170-nsnumber_bridge.md#alternatives-considered) declined to keep `NSNumber` as strongly type-preserving.

Reply:
We should likely just ignore this older API on Linux entirely and only support `ParameterEncoder`s.